Dockerfile

```
@@ -1,19 +1,13 @@
-RUN python -m pip install pip --upgrade
-WORKDIR /app/fast-pysf
-ADD ./fast-pysf/requirements.txt .
-RUN python -m pip install -r requirements.txt
-ADD ./fast-pysf .
-RUN python -m pip install .
+RUN python -m pip install uv
 WORKDIR /app
-ADD ./requirements.txt .
-RUN python -m pip install -r requirements.txt
-ADD ./setup.py ./setup.py
+ADD ./uv.toml ./uv.lock ./pyproject.toml ./
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)